BuildResidentialHPXML/measure.rb

```diff
@@ -2897,6 +2897,12 @@ def arguments(model)
       args << arg
     end
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('apply_validation', false)
```

Comment:
Can we make `apply_defaults` consistent with this new `apply_validation` argument? I'm thinking in terms of position within the arguments section (i.e., at the bottom), as well as display name and description formatting.

Reply:
Done.